src/mutability/Mutator.sol

```diff
@@ -48,13 +66,19 @@ contract Mutator is IMutator, Derived, Pausable {
         _nameToMutable[name].upgrade{value: msg.value}(implementation, data);
     }
 
-    function _pause() internal override {
+    function _pause() internal {
```

Comment:
I suppose this was a problem before, but what happens if we (1) pause, then (2) create a new mutable, then (3) unpause.

I think step (3) would revert since the new mutable would not be in then "unpaused" state.

Reply:
Agree; two options:
- Update `Mutator.create` to check the state of the first _Mutable_ in the array (if any).  Pause after creation if appropriate.  This should ensure consistent state.
- Update `Mutator.pause` and `unpause` to check state in each iteration.  This seems safer but the cost would exascerbate the unbounded iteration.

Reply:
Yeah, I kind of don't love either option because it would also require us to put a getter in the fallback of `Mutable` which we don't have structure for already.

I'm wondering if it'd be easier to just not delete `Pausable` (actually forgot it was being used here), and instead add a fix to the `create()` function to pause the new `Mutable` on creation if `Mutator` is paused.

Reply:
I like that.  Reverted the change in 1d558a6e1c580f6d1bd0480c9b9c64153dc20b09 and implemented pause-on-create in 9f1f9a60670a3e4589927722433ef5fce0a96a86.